vtex-apps/country-data-settings
# CHUI-50: Add address and phone configurations and GraphQL queries

#### What problem is this solving?

Initial version for this app. In this PR I added the schema for the country settings configuration, initially only with configuration for phone and address related fields (we are still brainstorming on how to validate documents with complex validation rules).

#### How should this be manually tested?

[Workspace](https://steps--checkoutio.myvtex.com/checkout/#/profile). In this workspace I'm using the country settings provided by this app in the phone field, where you can see that the Bolivian phone is now available :-)

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
